examples/practice-typescript/src/scripts/app.tsx

```diff
@@ -12,6 +12,9 @@ const editUser = () => console.log('edit');
 const deleteUser = () => console.log('delete');
 
 import TableController from 'components/TableController';
+import TableBody from 'components/TableBody';
+import { User } from 'mocks/user';
+import { Product } from 'mocks/product';
```

Comment:
The name here might confuse with type `Product`. We should have more predictable naming like `ProductListMock` if it is a list. Similar for `User` should be `UserMock` if it is single user

Reply:
Fixed